composer.json

```diff
@@ -29,5 +29,7 @@
             "SilverStripe\\Comments\\Tests\\": "tests/"
         }
     },
-    "license": "BSD-3-Clause"
+    "license": "BSD-3-Clause",
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

Comment:
these couple of lines only work in the root `composer.json` (https://getcomposer.org/doc/04-schema.md#minimum-stability)